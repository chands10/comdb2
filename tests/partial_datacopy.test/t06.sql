create table t {
    schema {
        int a
        int b
        int c
        int d
    }
    keys {
        datacopy(a, b, c) "ac" = a + c
    }
};$$


select * from comdb2_keys where tablename='t';
select * from comdb2_partial_datacopies where tablename='t';

drop table t;
