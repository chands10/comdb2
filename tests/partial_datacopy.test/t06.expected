(tablename='t', keyname='AC', keynumber=0, isunique='Y', isdatacopy='N', ispartialdatacopy='Y', isrecnum='N', condition=NULL, uniqnulls='N')
(tablename='t', keyname='AC', columnname='a')
(tablename='t', keyname='AC', columnname='b')
(tablename='t', keyname='AC', columnname='c')
